klayout/drc/run_drc.py

```diff
@@ -340,7 +340,13 @@ def check_klayout_version():
         exit(1)
     elif len(klayout_v_list) >= 2 and len(klayout_v_list) <= 3:
         if klayout_v_list[1] < 28:
-            logging.error("Prerequisites at a minimum: KLayout 0.28.0")
+            logging.error("Prerequisites at a minimum: KLayout 0.28.4")
+            logging.error(
+                "Using this klayout version has not been assesed in this development. Limits are unknown"
+            )
+            exit(1)
+        if klayout_v_list[1] == 28 and klayout_v_list[2] <= 3:
+            logging.error("Prerequisites at a minimum: KLayout 0.28.4")
```

Comment:
can't we combine the two checks?

Reply:
Is that okay ?

Reply:
@proppy I think we have agreed that those changes will be addressed later in #6 

Could you please focus in this PR on documentation changes? We have made some minor changes in the code to match the version number required for klayout.

Reply:
@atorkmabrains I agree with you (about focusing the PR on documentation), but this PR is adding a new version check (line 348 to 351); therefore I think it makes sense to ask why it is necessary and why it can't be achieved by changing the condition on line 341.

Reply:
Done